Cards and CSS created

AndrianaOS opened the pull request:
Cards created for each doctor, CSS included

AndrianaOS committed:
Cards and CSS created

AndrianaOS committed:
Added grid CSS to cards

AndrianaOS committed:
Included a picture and CSS

AndrianaOS committed:
Background colours changed

AndrianaOS committed:
Font colour changed

AndrianaOS committed:
Updated information for cards

Updated information for doctor cards and fetching from server instead of frontend array. Images still not appearing yet

AndrianaOS committed:
Change names to reflect DB

AndrianaOS committed:
Update DoctorCards.jsx

Add server address

Gayle-Thompson-Igwebike reviewed (commented):
We should play with the border for our images and see how we feel..

AndrianaOS committed:
Update App.js

Move fetch function to App.js from DoctorCards.jsx

AndrianaOS committed:
Remove and rename global name

AndrianaOS committed:
Update DoctorCards.css

Slight edit made to CSS. As server is down, section does not load for CSS to be completed

AndrianaOS committed:
Update DoctorCards.jsx

Move fetch function and useEffect to App.js. Remove commented out code

AndrianaOS committed:
Edit height and width of images
